build/configs/imxrt1050-evk/elf/defconfig

```diff
@@ -0,0 +1,896 @@
+# CONFIG_IMXRT_SEMC is not set
+# CONFIG_IMXRT_SNVS_LPSRTC is not set
+# CONFIG_IMXRT_SNVS_HPRTC is not set
+# CONFIG_IMXRT_USDHC is not set
+# CONFIG_IMXRT_GPIO_IRQ is not set
+
+#
+# Memory Configuration
+#
+CONFIG_IMXRT_BOOT_OCRAM=y
+CONFIG_IMXRT_OCRAM_PRIMARY=y
+
+#
+# i.MX RT Heap Configuration
+#
+
+#
+# Architecture Options
+#
+# CONFIG_ARCH_NOINTC is not set
+# CONFIG_ARCH_VECNOTIRQ is not set
+# CONFIG_ARCH_DMA is not set
+CONFIG_ARCH_HAVE_IRQPRIO=y
+# CONFIG_ARCH_L2CACHE is not set
+CONFIG_ARCH_HAVE_COHERENT_DCACHE=y
+# CONFIG_ARCH_HAVE_ADDRENV is not set
+# CONFIG_ARCH_NEED_ADDRENV_MAPPING is not set
+CONFIG_ARCH_HAVE_VFORK=y
+# CONFIG_ARCH_HAVE_MMU is not set
+CONFIG_ARCH_HAVE_MPU=y
+# CONFIG_ARCH_NAND_HWECC is not set
+# CONFIG_ARCH_HAVE_EXTCLK is not set
+# CONFIG_ARCH_HAVE_POWEROFF is not set
+CONFIG_ARCH_HAVE_RESET=y
+CONFIG_ARCH_USE_MPU=y
+# CONFIG_ARCH_IRQPRIO is not set
+CONFIG_ARCH_STACKDUMP=y
+# CONFIG_ENDIAN_BIG is not set
+# CONFIG_ARCH_IDLE_CUSTOM is not set
+CONFIG_ARCH_HAVE_RAMFUNCS=y
+# CONFIG_ARCH_RAMFUNCS is not set
+CONFIG_ARCH_HAVE_RAMVECTORS=y
+# CONFIG_ARCH_RAMVECTORS is not set
+
+#
+# Board Settings
+#
+CONFIG_BOARD_LOOPSPERMSEC=104926
+# CONFIG_ARCH_CALIBRATION is not set
+
+#
+# Interrupt options
+#
+CONFIG_ARCH_HAVE_INTERRUPTSTACK=y
+CONFIG_ARCH_INTERRUPTSTACK=0
+CONFIG_ARCH_HAVE_HIPRI_INTERRUPT=y
+# CONFIG_ARCH_HIPRI_INTERRUPT is not set
+
+#
+# Boot options
+#
+# CONFIG_BOOT_RUNFROMEXTSRAM is not set
+CONFIG_BOOT_RUNFROMFLASH=y
+# CONFIG_BOOT_RUNFROMISRAM is not set
+# CONFIG_BOOT_RUNFROMSDRAM is not set
+# CONFIG_BOOT_COPYTORAM is not set
+
+#
+# Boot Memory Configuration
+#
+CONFIG_RAM_REGIONx_START="0x20000000"
+CONFIG_RAM_REGIONx_SIZE="131072"
+# CONFIG_DDR is not set
+# CONFIG_ARCH_HAVE_SDRAM is not set
+
+#
+# Board Selection
+#
+CONFIG_ARCH_BOARD_IMXRT1050_EVK=y
+# CONFIG_ARCH_BOARD_ARTIK05X_FAMILY is not set
+# CONFIG_ARCH_BOARD_ESP32_FAMILY is not set
+CONFIG_ARCH_BOARD="imxrt1050-evk"
+
+#
+# Common Board Options
+#
+CONFIG_ARCH_HAVE_LEDS=y
+CONFIG_ARCH_LEDS=y
+CONFIG_ARCH_HAVE_BUTTONS=y
+# CONFIG_ARCH_BUTTONS is not set
+CONFIG_ARCH_HAVE_IRQBUTTONS=y
+# CONFIG_BOARD_CRASHDUMP is not set
+# CONFIG_LIB_BOARDCTL is not set
+# CONFIG_BOARD_FOTA_SUPPORT is not set
+
+#
+# Board-Specific Options
+#
+# CONFIG_IMXRT_NORFLASH is not set
+CONFIG_IMXRT_FLASH_PART=y
+CONFIG_IMXRT_FLASH_MINOR=0
+CONFIG_IMXRT_FLASH_PART_LIST="256,256,256,256,4096,"
+CONFIG_IMXRT_FLASH_PART_TYPE="none,none,ftl,ftl,smartfs,"
+CONFIG_IMXRT_AUTOMOUNT=y
+# CONFIG_IMXRT_AUTOMOUNT_SSSRW is not set
+CONFIG_IMXRT1050_EVK_HYPER_FLASH=y
+# CONFIG_IMXRT1050_EVK_NOR_FLASH is not set
+CONFIG_IMXRT_HYPERFLASH=y
+
+#
+# Board-Partition Options
+#
+# CONFIG_ARCH_USE_FLASH is not set
+
+#
+# Crypto Module
+#
+# CONFIG_CRYPTO is not set
+
+#
+# Kernel Features
+#
+# CONFIG_DISABLE_OS_API is not set
+
+#
+# Clocks and Timers
+#
+CONFIG_ARCH_HAVE_TICKLESS=y
+# CONFIG_SCHED_TICKLESS is not set
+CONFIG_USEC_PER_TICK=9979
```

Comment:
Most of the other imxrt1050-evk configs set this to 10000.
Please check this.

Reply:
ok. changed.